pkg/scaffold/ansible/galaxy_init.go

```diff
@@ -0,0 +1,42 @@
+package ansible
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/operator-framework/operator-sdk/pkg/scaffold/input"
+)
+
+// GalaxyInit - wrapper
+type GalaxyInit struct {
+	input.Input
+
+	Kind string
+	Dir  string
+}
+
+// GetInput - get input
+func (g *GalaxyInit) GetInput() (input.Input, error) {
+	if g.Path == "" {
+		dir, err := ioutil.TempDir("", "osdk")
```

Comment:
Most state derived at runtime should not be set in `GetInput()`. Move tmp dir creation and `g.Dir` setting responsibility to the caller.

Reply:
This should be removed if you're setting `Dir` in the caller. I'm not sure why `Dir` is necessary since its not a template pipeline and not used anywhere as far as I can tell.